ui/packages/consul-ui/tests/acceptance/dc/services/show/topology/stats.feature

```diff
@@ -0,0 +1,36 @@
+@setupApplicationTest
+Feature: dc / services / show / topology / stats
+  Scenario: Given metrics is disabled,the Topology tab should not display metrics
+    Given 1 datacenter model with the value "dc1"
+    And 1 node models
+    And 1 service model from yaml
+    ---
+    - Service:
+        Name: service-0
+        ID: service-0-with-id
+    ---
+    When I visit the service page for yaml
+    ---
+      dc: dc1
+      service: service-0
+    ---
+    And I see topology on the tabs
+    And I don't see the "[data-test-stats]" element
```

Comment:
I took the "easy way out" LOL. I started working on the page object but things got messy. This is done in the metrics feature file too.

Reply:
The "easy way" sometimes aka the pragmatic way! Thanks for explaining this here!